### ISSUE TITLE
make: INTERNAL: Exiting with 12 jobserver tokens available; should be 15!

### ISSUE THREAD

**Issue by dscho** [Alexpux/MSYS2-packages#232]
When trying to build MinGW packages using (MSys2) `make` with the `-j15` option, I frequently get this error (sometimes with different numbers):

```
make: INTERNAL: Exiting with 12 jobserver tokens available; should be 15!
```

Not sure whether this is a bug in `make` or in `msys-2.0.dll`...

**Comment by elieux** [comment]
Can you try to reproduce with Cygwin?

**Comment by Alexpux** [comment]
This is not error, just warning.

**Comment by dscho** [comment]
> Can you try to reproduce with Cygwin?

To be precise, this only happens when "cross" compiling for MinGW 32/64. So I guess I'll have to put `.../mingw64/bin` into the `PATH` before running the build.

**Comment by dscho** [comment]
> This is not error, just warning.

Incorrect. If that were so, I would not have reported the problem. As it is, the `make -j15 ... && <...>` fails to execute the latter commands, proving that the exit code of the `make` call is absolutely, definitely non-0.

**Comment by dscho** [comment]
Okay, hold off, I am testing with `ming32-make` and see that there might be more parallelization effects. Will keep you posted.

**Comment by dscho** [comment]
I am terribly sorry! The error log was so long that I missed that there has been a different issue (which I believe to be a bug, still in `make`'s parallel mode where multiple rules require the output of one central rule, and that output is somehow not available when *some* of those former rules are executed). The message I saw is indeed not the cause of a non-zero exit, in one of my tests, I got an exit code 0 but *still* saw this message.

So I will close this ticket and only reopen it if I manage to come up with a small and verifiable example to reproduce the issue I still have with the parallel build.
